### ISSUE TITLE
Finding Version skips next character

### ISSUE THREAD

**Issue by EasyAsABC123** [varchev/go-generic-artifactory-poller#2]
## Example

```
Package Name: Regulated-Servicing_develop
Repository Id: Servicing
Package Path: regulated-servicing_develop
Package Id: Regulated-Servicing_develop
```

The real artifact name is: Regulated-Servicing_v0.10.0.1.zip
Full Path: Servicing:regulated-servicing_develop/Regulated-Servicing_v0.10.0.1.zip

If I use the above it returns `v0.10.0.1`
If I change `Package Id: Regulated-Servicing_develop_` it returns `0.10.0.1`
If I change `Package Id: Regulated-Servicing_develop_v` it returns `.10.0.1`

I don't think the above is exactly right

**Comment by varchev** [comment]
You are right. The current behavior it to extract the version from the name of the artifact. The current implementation relies on the assumption that the user artifact file name would be named as follows:
package-id-as-defined-in-config-1.0.0.0.ext
In other words it relies on the fact that there the filename starts with the ID defined, followed by a single separating symbol (could be dot, dash, underscore, etc.), followed by the version and then the extension.  This is was the simplest possible approach that I found suitable for the purpose. As a next step I'm thinking of introducing an optional configuration that could define a more advances setting that would allow for more formats of the file name. This could be a regular expression, that the user would enter, which will tell the poller how to extract the version from the artifact name/path.
